src/github.com/matrix-org/dendrite/roomserver/storage/sql.go

```diff
@@ -372,6 +394,12 @@ func (s *statements) prepareEvents(db *sql.DB) (err error) {
 	if s.bulkSelectStateEventByIDStmt, err = db.Prepare(bulkSelectStateEventByIDSQL); err != nil {
 		return
 	}
+	if s.bulkSelectStateAtEventByIDStmt, err = db.Prepare(bulkSelectStateAtEventByIDSQL); err != nil {
```

Comment:
Random thought: Why aren't we inlining the SQL here? It's getting rather tedious to see `selectFooByBarSQL` and `selectFooByBarSQLStmt` when really we just need the `Stmt`. I don't think pulling them out to `consts` buys us anything, I don't find it more or less readable (just annoying as I have to CTRL+F the variable name to see what the actual SQL is).

WDYT?